salt/profiles/init.sls

```diff
@@ -65,6 +76,9 @@ profiles-uwsgi-service:
             - file: profiles-uwsgi-service
         - watch:
             - profiles-install
+            - profiles-app-config
+            - profiles-clients-config
+            - profiles-uwsgi-config
```

Comment:
I think this is currently redundant as `profiles-install` always returns a change, so it's restarted on every provision. But let's not rely on that.

Reply:
even `profiles-repository` could be in this list

Reply:
Might be able to watch the directory instead - https://github.com/saltstack/salt/issues/663